CGPs/cgp-0095.md

```diff
@@ -0,0 +1,100 @@
+
+Through this proposal, we aim to allocate the CELO-equivalent of $150k USD from the Celo Community Fund towards Centrifuge’s timely native deployment on Celo. 
+
+## Areas of Focus
+The scope of this proposal is to natively deploy and successfully launch Centrifuge on Celo mainnet by the end of October 2023. The associated $150k of development costs breaks out as follows:
+- $25k - Initial deployment cost of Centrifuge on Celo
+- $50k - Audit of Centrifuge on Celo
+- $75k - Support, upgrades and maintenance over the first 12 months
+
+## Success Criteria
+As outlined in CGP-84, “Total Debt Value Locked” is a key metric by which the Celo community can measure the success of Centrifuge. Several companies in the rapidly expanding Credit Collective have registered interest in utilizing Centrifuge on Celo to launch their debt pools.
+
+Since Centrifuge enables the issuance of “ReFi” and Natural Capital assets (eg. carbon credit forward contracts, renewable energy infrastructure loans, traditional development finance) - cumulative impact metrics around CO2, gigawatts, sustainable development goals (SDGs), and domain-specific parameters are relevant indicators of success. 
+
+An important feature of this deployment is that Centrifuge pools on Celo can accept capital from Ethereum mainnet and other chains. Consequently, it is expected that launching Centrifuge and associated debt pools will drive significant bridging volume towards Celo. 
+
+## Known Risks
+Given that Centrifuge is a standalone web3 protocol, there is minimal risk in deploying the infrastructure itself on Celo. The primary risks are associated with the possibility of defaults on debt pools, which is not linked to web3 but rather the underlying RWAs. This risk has been historically mitigated by the diligence conducted by the Centrifuge governance community in the “[Pool Onboarding Process (POP)](https://gov.centrifuge.io/t/introducing-the-pool-onboarding-proposal-pop/3846)”.
+
+The Centrifuge community has recently established the [Credit Group](https://gov.centrifuge.io/t/introducing-the-centrifuge-credit-group/4994), a team of outside experts capable of providing risk analysis for Pools at launch. This function helps to ensure that the underlying deal that establishes the Pool is arranged in a way that is consistent with best practices, competitive to market standards, and most importantly, with due regard to appropriate risk management and compensation for investment returns. 
+
+# Implementation
+## Budget
+We request to set our spend permit on the community fund at 400,000 CELO - but hope to not use the full amount. This number accounts for CELO at $0.375 (150,000 cUSD from 400,000 CELO) and protects the budget from acute market volatility. Any unutilized CELO will remain in the community fund.
+## Multisig
```

Comment:
Maybe you should use an average of the last 90 days for this calculation _"$0.375 (150,000 cUSD from 400,000 CELO)"_ because atm I just review this proposal Celo price is above $0,4

Reply:
From our side the language as written is accurate - ultimately the request is for $150k with a maximum of 400k CELO, and stipulation is included that "Any unutilized CELO will remain in the community fund"

Reply:
Note this proposal is phrased using precedent from past proposals (eg. https://github.com/celo-org/governance/blob/main/CGPs/cgp-0080.md#cusd-denominated-budget)